app/api/helpers/storage.py

```diff
@@ -149,7 +149,10 @@ def upload_local(uploaded_file, key, **kwargs):
     if not os.path.isdir(dir_path):
         os.makedirs(dir_path)
     uploaded_file.save(file_path)
-    return '/serve_' + file_relative_path
+    image_url = '/serve_' + file_relative_path
```

Comment:
Sorry. It is made in that way only. Forgot to remove serve_. Also, should it be `<staticdomain>/filename.png` or `<staticdomain>/static/filename.png`?